### PR TITLE
Add Skuratov Сoffee

### PR DESCRIPTION
**Pull request by afanasovich**

### CODE REVIEW

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -2401,6 +2401,21 @@
...
     },
+    {
+      "displayName": "Skuratov Сoffee",
+      "locationSet": {"include": ["ru"]},
+      "matchNames": ["Скуратов кофе"],
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Skuratov Сoffee",
+        "brand:en": "Skuratov Сoffee",
+        "brand:wikidata": "Q30891188",
```
Comment (arch0345) [inline data/brands/amenity/cafe.json:2412]: This looks like the wikidata page for Шоколадница, maybe this is a copy-paste mistake?

**Review by arch0345 — CHANGES_REQUESTED** [review]
The Wikidata page linked doesn't seem to match the brand

File: data/brands/amenity/cafe.json (hunk 1 above)
Comment (afanasovich) [reply data/brands/amenity/cafe.json:2412]: Thx.

I made a mistake when I copied the template from Шоколадница. 

Скуратов кофе — Q98642885 https://wikidata.org/wiki/Q98642885

**Review by arch0345 — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**